cirq-core/cirq/testing/consistent_decomposition.py

```diff
@@ -53,16 +53,22 @@ def _known_gate_with_no_decomposition(val: Any):
     """Checks whether `val` is a known gate with no default decomposition to default gateset."""
     if isinstance(val, ops.MatrixGate):
         return protocols.qid_shape(val) not in [(2,), (2,) * 2, (2,) * 3]
+    if isinstance(val, ops.ControlledGate):
```

Comment:
Flip if-block to reduce indentation:
```
if not isinstance(val, ops.ControlledGate):
    return False
# ...controlled gate checks...
```

Reply:
More known types without a decomposition can / will get added in future PRs, so the current structure works better.